sdk/servicebus/service-bus/samples/README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Install the library
 
-Run the below in your samples folder to install the npm package for Azure Service Bus library.
+Copy at least one of these files into a folder outside of this repository, then run the command below to install the npm package for the Azure Service Bus library.
```

Comment:
Lets talk about having a dedicated folder to run the samples and why the user cannot use the one in the repo. Then the step to install and then about copying the sample the user is interested in

Reply:
@ramya-rao-a 

Would something like the following be better?

---

## Working on a separate folder

Since these samples are inside of the repository of the NPM package `@azure/service-bus`, you won't be able to install `@azure/service-bus` inside of the samples folder. Instead, you'll need to copy at least one of the samples into a folder outside of this repository.

## Install the library

...
---

Thoughts?

Reply:
The `need to copy at least one of the samples` is not sitting well with me. They have to copy the sample that they are interested in. I would suggest to be as clear as possible.

## Install the library

- Choose a folder of your choice (outside this repository in case you have cloned it) to install the npm package and run your samples in. You cannot run the samples directly from the repository as we cannot install the package in the same folder that has the source code for it.
- In this chosen folder of yours, run the below to install the npm package
```shell
npm install @azure/service-bus 
```
- Now copy either all the samples or the sample of your choice to this folder

Reply:
Thank you! I'll update it 🌞

Reply:
@ramya-rao-a Hi! So, another option is to add a package.json in this folder instead, that would solve the recursive dependency thing. Thoughts?

Reply:
I'll make another PR :D

Reply:
Here it is: https://github.com/Azure/azure-sdk-for-js/pull/3118